docs/pages/docs.md

```diff
@@ -5,20 +5,58 @@ layout: default
 
 # Documentation
 
-## Doc building
-
-| Name                                            | Short description                                               | 🚦  |
-| ----------------------------------------------- | --------------------------------------------------------------- | :-: |
-| [sphinx](https://www.sphinx-doc.org/en/master/) | Generates documentation from reStructuredText or Markdown files | 🟢  |
-| [gitbook](https://www.gitbook.com/)             | General documentation builder; integrates with GitHub           | 🟠  |
-| [mkdocs](https://www.mkdocs.org/)               | Generates documentation from Markdown files                     | 🟠  |
-| [pdoc](https://pdoc.dev/)                       | Auto-generates API documentation from docstrings                | 🟠  |
-
-<details>
-<summary> 🟢 explanation</summary>
-Sphinx is the de-facto standard that is widely used. It is well tested, reliable and very customisable.
+With Python, as with many other languages, it's very common to automatically
+create a web page with documentation built from [docstrings] in your package's
+API. For your package to be easily reusable we recommend you build documentation
+pages and host them somewhere.
+
+If you're using GitHub, the simplest is to host your docs on [GitHub pages].
```

Comment:
I would disagree, and in my opinion it's simpler to use [readthedocs](https://readthedocs.org/) - would suggest we remove this in this PR, and defer documentation hosting dicsussion to a new issue?

Reply:
Oh! I never used RTD, but just saw you fighting with it for napari-matplotlib. It's what we do in the template. How about `s/simplest is to/one option is to/` ?

Reply:
```suggestion
If you're using GitHub, one option is to host your docs on [GitHub pages].
```

Reply:
👍  - `napari-matplotlib` was very much the outlier of "doesn't work with readthedocs" because we were trying to include 3D rendered images in the docs, and generate them as part of the build.